september/src/main/java/edu/asu/diging/gilesecosystem/september/core/model/IDataTableData.java

```diff
@@ -0,0 +1,23 @@
+package edu.asu.diging.gilesecosystem.september.core.model;
+
+import java.util.List;
+
+public interface IDataTableData {
```

Comment:
this does not look like a domain/model class and shouldn't be in the model package